File: Python/Problem-12.py
# Write a program to print all prime number in a intervial
# lower = int(input("Enter Any Number: "))
# upper = int(input("Enter Any Number: "))

# for i in range(lower , upper+1):
#   if i > 1:
#     for j in range(2, i):
#       if i % j == 0:
#         break
#     else:
#       print(i)

lower = int(input("Enter Any Number: "))
upper = int(input("Enter Any Number: "))

for num in range(lower , upper+1):
  if num > 1:
    for i in range(2 , num):
      if num % i == 0:
        break;
    else:
      print(num)

File: JavaScript/Problem-12.js
// # Write a program to print all prime number in a intervial
const prompt = require("prompt-sync")();
const is_Prime = (number) => {
  if (number <= 1) {
    return 0;
  } else {
    for (let i = 2; i <= Math.sqrt(number); i++) {
      if (number % i === 0) {
        return false;
      }
    }
    return true;
  }
};
const print_prime = (Up_To) => {
  console.log(`Print The Prime Number ${Up_To}`);
  for (let j = 2; j <= Up_To; j++) {
    if (is_Prime(j)) {
      console.log(j);
    }
  }
};
number = parseInt(prompt("Enter Any Number: "));
print_prime(number);
